pldoc/static/sass/_utilities.scss

```diff
@@ -34,6 +36,10 @@ $pldoc-ui-toolkit-color:            rgb(255, 222, 32);
     }
 }
 
+%grid-wrapper {
+    padding-right: spacing-horizontal(base);
+    padding-left: spacing-horizontal(base);
```

Comment:
Nit: The actual result would be the same, but it seems like elsewhere we do use the RTL mixin:

```
@include padding-right(spacing-horizontal(small));
@include padding-left(spacing-horizontal(small));
```